api/handlers/container/top.go

```diff
@@ -0,0 +1,156 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package container
+
+import (
+	"bytes"
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/containerd/containerd/v2/pkg/namespaces"
+	ncTypes "github.com/containerd/nerdctl/v2/pkg/api/types"
+	"github.com/containerd/nerdctl/v2/pkg/cmd/container"
+	"github.com/gorilla/mux"
+
+	"github.com/runfinch/finch-daemon/api/response"
+	"github.com/runfinch/finch-daemon/pkg/errdefs"
+)
+
+func (h *handler) top(w http.ResponseWriter, r *http.Request) {
+	cid, ok := mux.Vars(r)["id"]
+	if !ok || cid == "" {
+		response.JSON(w, http.StatusBadRequest, response.NewErrorFromMsg("must specify a container ID"))
+		return
+	}
+
+	psArgs := r.URL.Query().Get("ps_args")
+	if psArgs == "" {
+		// Set default ps arguments if none provided
+		psArgs = "-ef"
```

Comment:
What is nerdctl default?

Reply:
`"-ef"` is the default in nerdctl - https://github.com/containerd/nerdctl/blob/832c4556e0b82789f687b70d1e394b892e035722/pkg/cmd/container/top_unix.go#L54

Have removed this redundant code, we simply pass what user provides